Framework/Core/src/runDataProcessing.cxx

```diff
@@ -2388,36 +2393,36 @@ int doMain(int argc, char** argv, o2::framework::WorkflowSpec const& workflow,
+    ("start-port,p", bpo::value<unsigned short>()->default_value(22000), "start port to allocate")                                                                     //                                                                                                                     //
+    ("port-range,pr", bpo::value<unsigned short>()->default_value(1000), "ports in range")                                                                             //                                                                                                                       //
+    ("completion-policy,c", bpo::value<TerminationPolicy>(&processingPolicies.termination)->default_value(TerminationPolicy::QUIT),                                    //                                                                                                                       //
+     "what to do when processing is finished: quit, wait")                                                                                                             //                                                                                                                      //
+    ("error-policy", bpo::value<TerminationPolicy>(&processingPolicies.error)->default_value(TerminationPolicy::QUIT),                                                 //                                                                                                                          //
+     "what to do when a device has an error: quit, wait")                                                                                                              //                                                                                                                            //
+    ("min-failure-level", bpo::value<LogParsingHelpers::LogLevel>(&minFailureLevel)->default_value(LogParsingHelpers::LogLevel::Fatal),                                //                                                                                                                          //
+     "minimum message level which will be considered as fatal and exit with 1")                                                                                        //                                                                                                                            //
+    ("graphviz,g", bpo::value<bool>()->zero_tokens()->default_value(false), "produce graphviz output")                                                                 //                                                                                                                              //
+    ("mermaid", bpo::value<std::string>()->default_value(""), "produce graph output in mermaid format in file under specified name or on stdout if argument is \"-\"") //                                                                                                                              //
```

Comment:
```suggestion
    ("mermaid", bpo::value<std::string>()->default_value("-"), "produce graph output in mermaid format in file under specified name or on stdout if argument is \"-\"") //                                                                                                                              //
```

Reply:
Wouldn't that enable the `mermaid` mode when `--mermaid` is not specified ?

Reply:
ah, good point. Ok, then...